### DISCUSSION TITLE
Поддержка глобального FarMenu.ini не только в корне рядом с far2l, но и в ~/.config/far2l

### DISCUSSION THREAD

**Discussion by chronoxor** [elfmz/far2l#2184] [Ideas]
На MacOS нужно копировать FarMenu.ini в /Applications/far2l.app/Contents/MacOS и если нужно перенести конфиг, еще сходить туда и забрать FarMenu.ini. Было бы хорошо, если бы его можно было кинуть в основной конфиг ~/.config/far2l и оно тоже считалось бы глобальным и открывалось по F2.

**Comment by chronoxor** [comment]
Проверил в Far 3 там глобальный FarMenu.ini работает если его поместить в корень директории Profile, где лежат и другие конфиги

**Reply by shmuz** [reply]
`~/.config/far2l/settings/user_menu.ini` работает.

**Reply by chronoxor** [reply]
Да, но формат user menu отличается от FarMenu.ini...

**Comment by akruphi** [comment]
Отмечу, что формат `~/.config/far2l/settings/user_menu.ini` отличается от `FarMenu.ini` местных меню.

Спасибо за вопрос - немного поправил про это в помощи в #2185, надеюсь там теперь будет понятно без неоднозначностей.
